input/pages/index.md

```diff
@@ -0,0 +1,3304 @@
+
+To ensure deterministic evaluation, these operators should return the same value regardless of how many times they are evaluated within any given expression (i.e. now() should always return the same DateTime in a given expression, timeOfDay() should always return the same Time in a given expression, and today() should always return the same Date in a given expression.)
+
+##### now() : DateTime
+
+Returns the current date and time, including timezone offset.
+
+##### timeOfDay() : Time
+
+Returns the current time.
+
+##### today() : Date
+
+Returns the current date.
+
+<a name="definevariable"></a>
+#### defineVariable(name: String [, expr: expression])
+> Note: The contents of this section are Standard for Trial Use (STU)
+{: .stu-note }
+
+Defines a variable named `name` that is accessible in subsequent expressions and has the value of `expr` if present, otherwise the value of the input collection. In either case the function does not change the input and the output is the same as the input collection.
+{:.stu}
+
+If the name already exists in the current expression scope, the evaluation will end and signal an error to the calling environment. Note that functions that take an `expression` as an argument establish a scope for the iteration variables ($this and $index). If a variable is defined within such an expression, it is only available within that expression scope.
+{:.stu}
+
+
+Example:
+{:.stu}
+``` fhirpath
+group.select(
+  defineVariable('grp')
+  .select(
+    element.select(
+      defineVariable('src')
+      .target.select(
+        %grp.source & '#' & %src.code
+        & ' ' & equivalence & ' '
+        & %grp.target & '#' & code
+      )
+    )
+  )
+)
+```
+{:.stu}
+
+> Note: this would be implemented using expression scoping on the variable stack and after expression completion the temporary variable would be popped off the stack.
+{:.stu}
+
+#### lowBoundary([precision: Integer]): Decimal | Date | DateTime | Time
+{:.stu}
+
+The least possible value of the input to the specified precision.
+{:.stu}
+
+The function can only be used with Decimal, Date, DateTime, and Time values, and returns the same type as the value in the input collection.
+{:.stu}
+
+If no precision is specified, the greatest precision of the type of the input value is used (i.e. at least 8 for Decimal, 4 for Date, at least 17 for DateTime, and at least 9 for Time).
+{:.stu}
+
+If the precision is greater than the maximum possible precision of the implementation, the result is empty *(CQL returns null)*.
+{:.stu}
+
+If the input collection is empty, the result is empty.
+{:.stu}
+
+If the input collection contains multiple items, the evaluation of the expression will end and signal an error to the calling environment.
```

Comment:
I am not sure why this is necessary.  One might have a collection of Decimals as input.

Reply:
This isn't an implementation of min/max